packages/webcomponents/src/components/checkout/save-new-payment-method.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, h, State, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'justifi-save-new-payment-method',
+})
+export class SaveNewPaymentMethod {
+  @State() isChecked: boolean = false;
+  @Event({ bubbles: true }) checkboxChanged: EventEmitter<boolean>;
+
+  handleCheckboxChange(event: Event) {
+    const target = event.target as HTMLInputElement;
+    this.isChecked = target.checked;
+    this.checkboxChanged.emit(this.isChecked);
+  }
+
+  render() {
+    return (
+      <div class="mt-3 d-flex align-items-center">
```

Comment:
I tried with `mt-4` and it looked a little bit more centralized. WDYT?

mt-3: 
<img width="483" alt="image" src="https://github.com/user-attachments/assets/4e9bc7dd-f4af-4b83-9d5b-2ce2b87d5dcb">

mt-4: 
<img width="483" alt="image" src="https://github.com/user-attachments/assets/95c18c3a-acf5-4795-ab12-d3f3fec67e3f">

Reply:
Yeah, agreed. It's a little better with `mt-4`. I will update that.

Good catch!